src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1016,31 +1016,29 @@ private function registerRouterConfiguration(array $config, ContainerBuilder $co
                 ->replaceArgument(0, $config['default_uri']);
         }
 
-        if (\PHP_VERSION_ID >= 80000 || $this->annotationsConfigEnabled) {
```

Comment:
Doctrine annotation is meant here and not attributes, right? the if statement must remain.

Reply:
The annotation loader is always enabled on PHP 8. The change is correct.

Reply:
in Sf6, `\PHP_VERSION_ID >= 80000` is true. The if statement will always be true

Did I miss something?